docker/pypi/wmagent/init.sh

```diff
@@ -594,6 +626,13 @@ main(){
         echo "            docker kill wmagent"
         echo "          * Start a fresh instance of wmagent:"
         echo "            ./wmagent-docker-run.sh -t <WMA_TAG> && docker logs -f wmagent"
+        echo
+        echo "     d) If you are deploying inside a virtual environment"
```

Comment:
Where is c)? :)

Reply:
Not needed in the new Latin alphabet ;)
fixing it